airflow/providers/papermill/operators/papermill.py

```diff
@@ -33,6 +33,8 @@
 class NoteBook(File):
     """Jupyter notebook"""
 
+    template_fields: ClassVar = {*File.template_fields, "parameters"}
```

Comment:
```suggestion
    template_fields: ClassVar[str] = {*File.template_fields, "parameters"}
```

(I don’t think the annotation is actually needed at all though, it should inherit the type from `File`.)

Reply:
I guess you mean `ClassVar[Sequence[str]]` instead of just `ClassVar[str] `

However even with with this annotation I got

``` console

 error: Incompatible types in assignment (expression has type "Set[str]", variable has type "Sequence[str]")  [assignment]
```

So I change wrap value to tuple

Reply:
Can’t use sequence, you need Collection[str]. But changing it to tuple works as well, the difference is minimal.